client/my-sites/plans/current-plan/current-plan-thank-you/anti-spam-thank-you.tsx

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import React, { ReactElement } from 'react';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { Button } from '@automattic/components';
+import ThankYou, { ThankYouCtaType } from './thank-you';
+
+const ThankYouCta: ThankYouCtaType = ( { dismissUrl, recordThankYouClick } ) => {
+	const translate = useTranslate();
+	return (
+		<Button primary href={ dismissUrl } onClick={ () => recordThankYouClick( 'anti-spam' ) }>
+			{ translate( 'Go back to your site' ) }
+		</Button>
+	);
+};
+
+const AntiSpamProductThankYou = (): ReactElement => {
+	const translate = useTranslate();
+	return (
+		<ThankYou
+			illustration="/calypso/images/illustrations/thankYou.svg"
+			title={ translate( 'Say goodbye to spam!' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **22** times:
`translate( 'Say goodbye to spam' )` *ES Score: 13*